README.md

```diff
@@ -57,10 +57,61 @@ The structure should appear like so:
 └── readme.md
 ```
 
+## New Directory structure (effective July 10th 2017)
+
+Re-organization is being done to support upcoming profiles work and Data Plane swagger specs. The new structure will appear like below
+ 
+```bash
+.
+├──automation
```

Comment:
Not sure whether we intentionally skipped the top node `specification` or not https://github.com/Azure/azure-rest-api-specs/tree/reorg. Other than that LGTM! 👍